### PR TITLE
Fs3 600

### PR DESCRIPTION
**Pull request by kjose**

### CODE REVIEW

File: internal/apilogic/handle_request.go [hunk 1]
```diff
@@ -23,6 +24,11 @@ func BuildHandleRequest(req *http.Request) (*handle.Request, error) {
 		handleRequest.Mode = mode
 	}
 
+	// exposeAllKeys default true for flags route
+	if strings.Contains(req.URL.String(), "/flags") {
```
Comment (guillaumejacquart) [inline internal/apilogic/handle_request.go:28]: Hum, so if the url is `/flags?exposeAllKeys=toto`, I feel like the handleRequest.ExposeAllKeys flag will be false. Is that the expected behavior ?

Comment (guillaumejacquart) [reply internal/apilogic/handle_request.go:28]: Also, I guess you thought about it, but there was no way to do it without hardcoding this "if", right ?

Comment (kjose) [reply internal/apilogic/handle_request.go:28]: It is possible I I refactor the BuildHandleRequest method, car for the moment il takes only the Request as an argument and golang does not allow to update the request from the controller :/

Comment (kjose) [reply internal/apilogic/handle_request.go:28]: If `/flags?exposeAllKeys=toto`, yes it will be false but it should not be implemented also ?

**Pull request merged**